src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx

```diff
@@ -117,6 +130,9 @@ class EventDetailsContent extends AsyncComponent<Props, State & AsyncComponent['
           />
           <EventHeader event={event} />
           <Controller>
+            <StyledButton size="small" onClick={this.toggleSidebar}>
+              {isSidebarVisible ? 'Hide Details' : 'Show Details'}
```

Comment:
```suggestion
              {isSidebarVisible ? t('Hide Details') : t('Show Details')}
```

Reply:
Crap I just saw this - will do it in a new branch